transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/LegacyFlgDestination.java

```diff
@@ -15,28 +15,43 @@
 package com.google.android.datatransport.cct;
 
 import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import com.google.android.datatransport.runtime.Destination;
 import java.io.UnsupportedEncodingException;
 
 public class LegacyFlgDestination implements Destination {
   static final String DESTINATION_NAME = "lflg";
   private static final String DEFAULT_API_KEY =
       StringMerger.mergeStrings("AzSCki82AwsLzKd5O8zo", "IayckHiZRO1EFl1aGoK");
+  private static final String EXTRAS_VERSION_MARKER = "lfdv1$";
```

Comment:
might make sense to drop the prefix?
```suggestion
  private static final String EXTRAS_VERSION_MARKER = "1$";
```